api/bin/spark.js

```diff
@@ -73,16 +74,21 @@ const logger = {
   request: ['1', 'true'].includes(REQUEST_LOGGING) ? console.info : () => { }
 }
 
-const handler = await createHandler({
-  client,
+// Initialize Fastify app
+const fastifyApp = await createFastifyApp({
+  pgPool,
```

Comment:
please update `logger` according to https://github.com/CheckerNetwork/spark-stats/blob/2558a45d7d17dd10a8671cfe8934ddfd55cf4363/stats/bin/spark-stats.js#L17-L19